test/services/notifications/setup/submit-check.service.test.js

```diff
@@ -59,7 +64,33 @@ describe('Notifications Setup - Submit Check service', () => {
   it('correctly triggers the "DetermineRecipientsService"', async () => {
     await SubmitCheckService.go(session.id)
 
-    expect(DetermineRecipientsService.go.calledWith([recipients.primaryUser])).to.be.true()
+    expect(DetermineRecipientsService.go.calledWith(testRecipients)).to.be.true()
+  })
+
+  it('correctly triggers the "CreateEventService"', async () => {
```

Comment:
I think this will be a temporary test whilst the rest of the code is developed. 

I think when the times is right there is a benefit to having this do the real calls to the DB and go through all the code (bar notify call).